TypeProvider/Core/Naming.fs

```diff
@@ -0,0 +1,30 @@
+﻿[<RequireQualifiedAccess>]
+module internal Froto.TypeProvider.Core.Naming
```

Comment:
This would make sense, indeed, if Froto is also supposed to generate C# classes via Roslyn API. Another thing that might be useful is to have mappings from protobuf scalar types to .NET types in some common place as well.